### ISSUE TITLE
How to enable multi-gpu training in the code?

### ISSUE THREAD

**Issue by DeepRunner** [KunpengLi1994/VSRN#9]
Hi, I wonder how to  enable multi-gpu training in the code?  And how much about a typical training time?
Thanks!

**Comment by KunpengLi1994** [comment]
Hi,

We did not implement the code to support multi-GPU. I should take around 1 day for training on MS-COCO using one single TITAN XP GPU

**Comment by DeepRunner** [comment]
Thanks!  And how many num of epochs do you usually need to achieve best performance? 30 epochs?

**Comment by KunpengLi1994** [comment]
Yes and we follow VSE++ to store the model with best performance on the validation set.

**Comment by KunpengLi1994** [comment]
Actually, we just simply follow the same training procedure as well as parameter settings of VSE++ and maybe this part can be further optimized or tuned eg. learning rate, epochs model selection etc.
